api/src/main/java/jakarta/data/page/Page.java

```diff
@@ -41,6 +41,13 @@
  */
 public interface Page<T> extends Slice<T> {
 
+    /**
+     * Returns {@code true} if it is known that there are more results, so that
+     * {@link #nextPageRequest()} will definitely not return {@code null}.
+     * @return {@code false} if this the last page of results
```

Comment:
```suggestion
     * Returns {@code true} if it is known that there are more results or that it is
     * necessary to request a next page to determine whether there are more results, so that
     * {@link #nextPageRequest()} will definitely not return {@code null}.
     * @return {@code false} if this is the last page of results.
```